packages/babel-types/src/definitions/flow.js

```diff
@@ -250,15 +251,16 @@ defineType("NumberTypeAnnotation", {
 });
 
 defineType("ObjectTypeAnnotation", {
-  visitor: ["properties", "indexers", "callProperties"],
+  visitor: ["properties", "indexers", "callProperties", "builder"],
   aliases: ["Flow", "FlowType"],
+  builder: ["exact"],
   fields: {
     properties: validate(
       arrayOfType(["ObjectTypeProperty", "ObjectTypeSpreadProperty"]),
     ),
     indexers: validateOptional(arrayOfType("ObjectTypeIndexer")),
     callProperties: validateOptional(arrayOfType("ObjectTypeCallProperty")),
-    exact: validate(assertValueType("boolean")),
+    exact: validateDefault(assertValueType("boolean"), false),
```

Comment:
I'd prefer this to be written as an object instead of using the `validateDefault` function, since `validator` and `default` have very little in common.